### PR TITLE
Learn blockquote component

### PR DESCRIPTION
**Pull request by chasulin**

### CODE REVIEW

File: src/screens/Learn/Learn.style.tsx [hunk 1]
```diff
@@ -45,7 +33,50 @@ export const PageHeader = styled.h1`
...
+
+  p {
+    &:not(:last-child) {
+      margin-bottom: 1rem;
+    }
+  }
+`;
+
+const BlockquoteStyles = css`
+  background-color: #fafafa;
```
Comment (pnavarrc) [inline src/screens/Learn/Learn.style.tsx:64]: Maybe we should name this color, we have a couple of instances of it already

File: src/screens/Learn/Learn.style.tsx [hunk 2]
```diff
@@ -45,7 +33,50 @@ export const PageHeader = styled.h1`
...
+  padding: 1.125rem;
```
Comment (pnavarrc) [inline src/screens/Learn/Learn.style.tsx:65]: Consider using spacing from the theme (either rounding to 16 or 24). I'm actually not sure what's best here, using `rem` or theme spacing 🤔

File: src/screens/Learn/Learn.style.tsx [hunk 3]
```diff
@@ -45,7 +33,50 @@ export const PageHeader = styled.h1`
...
+  display: inline-block;
```
Comment (pnavarrc) [inline src/screens/Learn/Learn.style.tsx:66]: Why `inline-block` instead of block?

**Review by pnavarrc — APPROVED** [review]
Looks good!

File: src/screens/Learn/Learn.style.tsx (hunk 2 above)
Comment (chasulin) [reply src/screens/Learn/Learn.style.tsx:65]: the mocks had 20px of spacing so i kept it exact, but i can round down to 16 and use theme spacing

File: src/screens/Learn/Learn.style.tsx (hunk 3 above)
Comment (chasulin) [reply src/screens/Learn/Learn.style.tsx:66]: the small top/bottom margin i was adding on the blockquote wasn't working with `display: block`

**Pull request merged**